mollie/organizations_test.go

```diff
@@ -0,0 +1,59 @@
+package mollie
```

Comment:
You are only testing the happy path, can you please also add some tests for the errors returned?

If you don't know how to do it, probably is worth to take a look at the tests for `payments`